src/tests/intg/getsockopt_wrapper.c

```diff
@@ -87,11 +87,15 @@ static void fake_peer_uid_gid(uid_t *uid, gid_t *gid)
     val = getenv("SSSD_INTG_PEER_UID");
     if (val != NULL) {
         *uid = atoi(val);
+    } else {
+        *uid = 0;
```

Comment:
Hi,

why `0`, wouldn't it be better to use `-1` or id of `nobody` or return an error if the environment variables are not set?

bye,
Sumit

Reply:
I guess I was lazy to set `SSSD_INTG_PEER_UID` in intg-tests and decided to "default to 0" (IIUC, intg-tests are being fully replaced by 'system' tests).
Ok, I'll fix this (will set env var in the test).

Reply:
Fixed.

Reply:
Thanks